src/consensus/pbft/libbyz/Big_req_table.cpp

```diff
@@ -223,6 +223,8 @@ bool Big_req_table::add_unmatched(Request* r, Request*& old_req)
     centry.requests.front()->request_id() >= r->request_id())
   {
     // client is expected to send requests in request id order
+    LOG_FAIL << "client is expected to send requests in request id order"
+             << r->client_id() << std::endl;
```

Comment:
`LOG_FAIL_FMT("Requests are not in request id order from client: {}", r->client_id());`